_variables.yml

```diff
@@ -1,6 +1,6 @@
 status:
-    done: '[{{< fa circle-check size="xl" title="Completed an MVP" >}}]{style="color: #48DC76"} *Completed*'
-    wip: '[{{< fa hammer size="xl" title="In progress" >}}]{style="color: #32A255"} *In progress*'
-    planned: '[{{< fa map size="xl" title="Planned" >}}]{style="color: #ADADAD"} *Planned*'
-    potential: '[{{< fa circle-question size="xl" title="Potential" >}}]{style="color: #D6D6D6"} *Potential*'
-    ongoing: '[{{< fa repeat size="xl" title="Ongoing" >}}]{style="color: #32A255"} *Ongoing*'
+    done: '[{{< fa circle-check size="xl" title="Completed an MVP" >}}]{style="color: #48DC76"}<br><small>*Completed*</small>'
+    wip: '[{{< fa hammer size="xl" title="In progress" >}}]{style="color: #32A255"}<br><small>*In progress*</small>'
+    planned: '[{{< fa map size="xl" title="Planned" >}}]{style="color: #ADADAD"}<br><small>*Planned*</small>'
+    potential: '[{{< fa circle-question size="xl" title="Potential" >}}]{style="color: #D6D6D6"}<br><small>*Potential*</small>'
+    ongoing: '[{{< fa repeat size="xl" title="Ongoing" >}}]{style="color: #32A255"}<br><small>*Ongoing*</small>'
```

Comment:
My only thought was that Ongoing and Completed should have the same colour, as the Ongoing items are in a state of completion every time we've added to it. But it isn't a hill I'll die on ;-)

Reply:
My thought was that something "ongoing" could be seen as being "in progress", since we're still working on it. And its not truly "done" as we define it in the roadmap with an MVP :) 
The colours are not a part of this PR, but #73